gui/src/i18n/vi/translation.json

```diff
@@ -0,0 +1,460 @@
+    "RIGHT_HAND": "Tay phải",
+    "RIGHT_UPPER_LEG": "Chân phải trên",
+    "RIGHT_LOWER_LEG": "chân phải dưới",
+    "RIGHT_FOOT": "Bàn chân phải",
+    "CHEST": "Ngực",
+    "WAIST": "Eo",
+    "HIP": "Hông",
+    "LEFT_SHOULDER": "Vai trái",
+    "LEFT_UPPER_ARM": "Tay trái trên",
+    "LEFT_LOWER_ARM": "tay trái dưới",
+    "LEFT_HAND": "Tay trái",
+    "LEFT_UPPER_LEG": "Chân trái trên",
+    "LEFT_LOWER_LEG": "Chân trái dưới",
+    "LEFT_FOOT": "Bàn chân trái"
+  },
+  "skeleton-bone": {
+    "NONE": "Chưa liên kết",
+    "HEAD": "Đầu ca",
+    "NECK": "Cổ dài",
+    "TORSO": "Chiều dài thân",
+    "CHEST": "Khoảng cách ngực",
+    "WAIST": "Khoảng cách eo",
+    "HIP_OFFSET": "Lệc đo hông",
+    "HIPS_WIDTH": "Chiều rộng hông",
+    "LEGS_LENGTH": "Chiều dài chân",
+    "KNEE_HEIGHT": "Chiều cao đầu gối",
+    "FOOT_LENGTH": "CHiều dài bàn chân",
+    "FOOT_SHIFT": "Lệch đo bàn chân",
+    "SKELETON_OFFSET": "Lệch đo thân",
+    "CONTROLLER_DISTANCE_Z": "Khoảng cách tay cầm Z",
+    "CONTROLLER_DISTANCE_Y": "Khoảng cách tay cầm Y",
+    "FOREARM_LENGTH": "khoảng cách cánh tay",
+    "SHOULDERS_DISTANCE": "Khoảng cách vai",
+    "SHOULDERS_WIDTH": "Chiều rộng vai",
+    "UPPER_ARM_LENGTH": "Chiều dài tay trên",
+    "ELBOW_OFFSET": "Lệch đo khuỷu tay"
+  },
+  "reset": {
+    "reset-all": "Reset tất cả bộ phận",
+    "full": "Reset",
+    "mounting": "Reset vị trí cài",
+    "quick": "Reset nhanh"
+  },
+  "serial-detection": {
+    "new-device.p0": "New serial device detected!" ,
+    "new-device.p1": "Vui lòng nhập thông tin Wi-Fi!",
+    "new-device.p2": "Vui lòng chọn những gì làm với nó",
+    "open-wifi": "Kết nối Wi-Fi",
+    "open-serial": "Mở Serial Console",
+    "submit": "Đăng lên!",
+    "close": "Đóng"
+  },
+  "navbar": {
+    "home": "Home",
+    "body-proportions": "Bộ phận cơ thể",
+    "trackers-assign": "Giao bộ phận tracker",
+    "mounting": "Đo vị trí đặt",
+    "onboarding": "Trình hướng dẫn cài đặt",
+    "settings": "Cài đặt"
+  },
+  "bvh": {
+    "start-recording": "Ghi BVH",
+    "recording": "Đang ghi..."
+  },
+  "overlay": {
+    "is-visible-label": "Xem overlay trên SteamVR",
+    "is-mirrored-label": "Xem overlay trong gương"
+  },
+  "tracker": {
+    "status": {
+      "none": "Không có tình trạng",
+      "busy": "Bận",
+      "error": "Lỗi",
+      "disconnected": "Đã ngắt kết nối",
+      "occluded": "Bị tắc",
+      "ok": "Đã kết nối"
+    },
+    "table": {
+      "column": {
+        "name": "Tên",
+        "type": "Loại",
+        "battery": "Pin",
+        "ping": "Ping",
+        "rotation": "Chiều chuyển X/Y/Z",
+        "position": "Vị trí X/Y/Z",
+        "url": "URL"
+      }
+    },
+    "rotation": {
+      "front": "Đằng trước",
+      "left": "Bên trái",
+      "right": "Bên phải",
+      "back": "Đằng sau" 
+    },
+    "infos": {
+      "manufacturer": "Nhà sản xuất",
+      "display-name": "Tên gọi",
+      "custom-name": "Tên tự chọn",
+      "url": "Tacker URL"
```

Comment:
Tacker should be Tracker
Also is there not a translation for "Tracker"? It seems you didn't translate it anywhere.

Reply:
Right, about that I could translate tracker but Vietnamese especially young would be more associated with word tracker.